frontend/src/composables/useRpcUrls.ts

```diff
@@ -0,0 +1,8 @@
+import { ChainConfigMapping } from '@/types/config';
+
+export function useRpcUrls(chains: ChainConfigMapping) {
```

Comment:
This is not a composable. Composables encapsulate stateful logic and return some functions and/or references.
This looks like a helper function to me.